examples/browser.js

```diff
@@ -1,146 +1,176 @@
 /* eslint-disable */
 
-const hostname = "metis.mapd.com"
-const protocol = "https"
-const port = "443"
-const database = "mapd"
-const username = "mapd"
-const password = "HyperInteractive"
-
-// The total number of tweets from Columbia
-const query = "SELECT count(*) AS n FROM tweets_nov_feb WHERE country='CO'"
-// try changing airtime to arrdelay in the query
-const query2 =
-  "SELECT carrier_name as key0, AVG(airtime) AS val FROM flights_donotmodify WHERE airtime IS NOT NULL GROUP BY key0 ORDER BY val DESC LIMIT 100"
+window.connection = null
+const autologin = false
+const sample_query =
+  "SELECT name, ST_Area(omnisci_geo) as val FROM omnisci_counties;"
 const defaultQueryOptions = {}
-const connector = new window.MapdCon()
-
-connector
-  .protocol(protocol)
-  .host(hostname)
-  .port(port)
-  .dbName(database)
-  .user(username)
-  .password(password)
-  .connectAsync()
-  .then(session =>
-    // now that we have a session open we can make some db calls:
-    Promise.all([
-      session.getTablesAsync(),
-      session.getFieldsAsync("flights_donotmodify"),
-      session.queryAsync(query, defaultQueryOptions),
-      session.queryAsync(query2, defaultQueryOptions)
-    ])
-  )
-  // values is an array of results from all the promises above
-  .then(values => {
-    // handle result of getTablesAsync
-    console.log(
-      `All dashboards available at ${hostname}:\n`,
-      values[0].map(x => x.name)
-    )
-
-    // handle result of getFieldsAsync
-    console.log(
-      "All fields for 'flights_donotmodify':",
-      values[1].columns.reduce((o, x) => Object.assign(o, { [x.name]: x }), {})
-    )
-
-    // handle result of first query
-    document.getElementById("result-async").innerHTML =
-      "There are " + values[2][0].n + " tweets from Columbia."
-    console.log("Query 1 results:", Number(values[2][0].n))
-
-    // handle result of 2nd query
-    createRowChart(values[3])
-    console.log(
-      "Query 2 results:",
-      values[3].reduce((o, x) => Object.assign(o, { [x.key0]: x.val }), {})
-    )
-  })
-  .catch(error => {
-    console.error("Something bad happened: ", error)
-  })
+const defaultConnection = {
+  hostname: "localhost",
+  useHTTPS: true,
+  port: "6278",
+  database: "omnisci",
+  username: "admin",
+  password: "HyperInteractive"
+};
```

Comment:
@jclay What if we wait on merging this PR until after https://github.com/omnisci/mapd-connector/pull/211 ? Then we could use the metis creds so that it still works out of the box.